sparklightautoml/automl/presets/tabular_presets.py

```diff
@@ -285,20 +295,42 @@ def get_gbms(
             time_score = self.get_time_score(n_level, key)
             gbm_timer = self.timer.get_task_timer(algo_key, time_score)
             if algo_key == "lgb":
```

Comment:
consider incapsulating logic of checking `algo_key` into separate function/class